test/jdk/tools/jpackage/windows/WinLongVersionTest.java

```diff
@@ -66,19 +66,6 @@
  *    scenario must be uninstalled
  */
 
-/*
- * @test
- * @summary jpackage with long version number
- * @library /test/jdk/tools/jpackage/helpers
- * @key jpackagePlatformPackage
- * @requires (jpackage.test.SQETest != null)
```

Comment:
Do you know if SQE actually was testing this test? Duplicated annotation is different only in `jpackage.test.SQETest`.

Reply:
No, they didn't. I checked the test spec, it doesn't reference this test. That is why I deleted this